AppLand/scss/_variables.scss

```diff
@@ -0,0 +1,26 @@
+/*========================================================================================
+*********   ALL OF THE VARIABLES TYPE 
+*******     COLORS 
+*****       GRIDS
+***         
+*/
+
+/*========================================================================================*\
+*----------------------------               COLORE              ----------------------------        
+\*========================================================================================*/
+
+    /*--------------------------------
+    *** FLAT UI COLORS 
+    **  flatuicolors.com/palette/defo
+    */
+
+    $BLACK          : rgb(000, 000, 000); // #000
+    $WHITE          : rgb(255, 255, 255); // #fff
+
+    $ShadowBlack    : rgb(024, 026, 027); // #181a1b
+    $VibrantGreen   : rgb(084, 234, 084); // #54ea54
+    
+    $LimeGreen      : rgb(071, 198, 071); // #47c647
+    $LightGray      : rgb(170, 170, 170); // #aaa
+
+    $TransparentBlack : rgba(0, 0, 0, 0.5); // shadow Black
```

Comment:
please, follow the code convention rules

Reply:
How should I write?

Reply:
read code convention, if you don't know ask the team